<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainPage</class>
 <widget class="QWidget" name="mainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>108</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>108</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DatasetTool</string>
  </property>
  <widget class="QPushButton" name="pb_move_steps">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mover pasos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_move_meters">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mover metros</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_steps">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_meters">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_up">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Arriba</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_down">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abajo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_left">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izquierda</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_right">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Derecha</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
